.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - /^\d+\.\d+\.\d(-alpha\.\d+|-beta\.\d+|-rc\.\d+)?$/
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Test, Build & Release
+    runs-on: "ubuntu-latest"
+    env:
+      GH_ACTIONS_NODE_VERSION: "10"
+      LOGS_DIR: /tmp/code-style/logs
+      LOGS_FILE: /tmp/code-style/logs/build-perf.log
+      TZ: "Europe/Brussels"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 10
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10"
+
+      - name: Create file & folder for GitHub Actions logs
+        run: |
+          # cfr scripts/_ghactions-group.sh
+          mkdir -p $LOGS_DIR
+          touch $LOGS_FILE
+
+      - name: Get tag name if exists
+        id: get_tag_name
+        run: echo ::set-output name=TAG_NAME::$(echo $GITHUB_REF | sed -n "s/^refs\/tags\/\(\S*\).*$/\1/p")
+
+      - name: List main variables
+        run: |
+          echo "Commit SHA  : ${GITHUB_SHA}"
+          echo "Tag         : ${GH_ACTIONS_TAG}"
+          echo "Reference   : ${GITHUB_REF}"
+          echo "Repository  : ${GITHUB_REPOSITORY}"
+          echo "Event       : ${GITHUB_EVENT_NAME}"
+          echo "Author      : ${GITHUB_ACTOR}"
+          NODE_VERSION="$(node -v)"
+          echo "Node version: $NODE_VERSION"
+          npm i -g npm@6.9.2
+          # This ensures that we are authenticated without requiring to have an actual .npmrc file within the project
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" >> ~/.npmrc
+        env:
+          GH_ACTIONS_TAG: ${{ steps.get_tag_name.outputs.TAG_NAME }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Build
+        run: npm run build
+
+      - uses: actions/upload-artifact@v1
```

Comment:
Can you add a comment above this line about where we can find info over `actions/upload-artifact@v1`?